demo/assets/head/license.txt

```diff
@@ -0,0 +1,11 @@
+Model Information:
```

Comment:
have you considered the licenses for the other models as well?

Reply:
Do we have to write this in the report or somewhere related? (Also for the other models?) (As we'll include screenshots of the demo application)